lib/capistrano/tasks/magento.rake

```diff
@@ -309,8 +309,15 @@ namespace :magento do
           end
 
           # Run again with HTTPS env var set to 'on' to pre-generate secure versions of RequireJS configs
-          deploy_flags = ['javascript', 'css', 'less', 'images', 'fonts', 'html', 'misc', 'html-minify']
-            .join(' --no-').prepend(' --no-');
+          deploy_flags = ['css', 'less', 'images', 'fonts', 'html', 'misc', 'html-minify']
+          
+          # As of Magento 2.1.3, it became necessary to exclude "--no-javacript" in order for secure versions of 
+          # RequireJs configs to be generated
+          if _magento_version <= Gem::Version.new('2.1.3')
```

Comment:
This is incorrectly using the <= operator. As it is, the flag will still be used on version 2.1.3. Can you fix this?